### DISCUSSION TITLE
Implement dark theme

### DISCUSSION THREAD

**Discussion by PritamSarbajna** [PritamSarbajna/tourism-website#13] [Ideas]
Hey this thread is for implementing dark theme for the website. Share your ideas or any image link you think fit for it.

**Comment by Iamslok** [comment]
![screenshot](https://user-images.githubusercontent.com/90669886/184546057-e3003b74-a707-439b-a590-ee1fe8a159f2.png)
@PritamSarbajna Demo

**Reply by PritamSarbajna** [reply]
No @Iamslok I already told you I don't want pure black theme

**Reply by PritamSarbajna** [reply]
What do you think about [this](https://www.pexels.com/video/an-aerial-footage-of-the-tokyo-tower-4711694/) as a video background image ???

**Reply by Iamslok** [reply]
> What do you think about [this](https://www.pexels.com/video/an-aerial-footage-of-the-tokyo-tower-4711694/) as a video background image ???

Yeah @PritamSarbajna it's really awesome

**Comment by PritamSarbajna** [comment]
Hey @Iamslok just let me create a rough layout so that you can understand how I want to look this website as

**Reply by Iamslok** [reply]
Yeah @PritamSarbajna sure

**Comment by PritamSarbajna** [comment]
Hey @Iamslok, see this is a rough idea but don't make similar. Is it looking good ?

https://user-images.githubusercontent.com/90236635/184549103-2e47f74e-d28a-4cc8-bcf7-74383bdca004.mp4

**Reply by Iamslok** [reply]
@PritamSarbajna it's looking good, but the dark is too shady.

**Reply by PritamSarbajna** [reply]
Yeah @Iamslok it's too shady how about deep blue ?

**Reply by Iamslok** [reply]
Yeah @PritamSarbajna it would be nice

**Reply by PritamSarbajna** [reply]
So it's looking like this and I didn't like it much. Let me know what do you think ?? optionally you can go [here](https://colorhunt.co/palettes/dark) to checkout other color combinations in a dark theme.

https://user-images.githubusercontent.com/90236635/184576514-0dcf00bd-2398-4758-9db1-44a4336121c4.mp4

**Reply by Iamslok** [reply]
@PritamSarbajna it's looking good, i think it's better looking than the darker onces.

**Reply by Iamslok** [reply]
> So it's looking like this and I didn't like it much. Let me know what do you think ?? optionally you can go [here](https://colorhunt.co/palettes/dark) to checkout other color combinations in a dark theme.
> 
> https://user-images.githubusercontent.com/90236635/184576514-0dcf00bd-2398-4758-9db1-44a4336121c4.mp4
> 
> 
> 

@PritamSarbajna The reason of its looking little bit odd, because of the font and it's color, the contactus page pink colour and the social icons design. We need to upgrade all the fonts and it's color and some designs.

**Reply by PritamSarbajna** [reply]
Yeah man I already raised these issues I can resolve this in a minute but I handed over to the contributors. So we have to wait or you can resolve this if the issues are not already work in progress

**Reply by Iamslok** [reply]
@PritamSarbajna can you suggest me some fonts?

**Reply by PritamSarbajna** [reply]
@Iamslok just use any basic fonts like ariel, roboto, calibri also give it a try with coolvetica see which of these is fit for it.

**Reply by PritamSarbajna** [reply]
How is it going @Iamslok ?

**Reply by Iamslok** [reply]
Yeah @PritamSarbajna it's going well tomorrow I'll show the demo

**Comment by PreciousOlamide** [comment]
hello, I am interested in working on this task

**Reply by PritamSarbajna** [reply]
Hey @PreciousOlamide you and @Iamslok also collaborate for this feature

**Reply by PreciousOlamide** [reply]
Okay, I will get started on  it

**Comment by PritamSarbajna** [comment]
Thank you @prakhartiwari2507  🙌
